content/_includes/NetworkBridgeSCALE.md

```diff
@@ -4,19 +4,19 @@
 To set up a bridge interface, from the **Network** screen:
 
 1. Click **Add** in the **Interfaces** widget. The **Add Interface** configuration screen displays.
- 
+
 ![AddInterfaceNicDeviceSCALE](/images/SCALE/22.12/AddInterfaceNicDeviceSCALE.png "Add Bridge Interface")
 
 2. Select **Bridge** from the **Type** dropdown list. You cannot change the **Type** field value after you click **Save**.
 
 3. Enter a name for the interface. Use the format *bondX*, *vlanX*, or *brX* where *X* is a number representing a non-parent interface. Read-only when editing an interface. You cannot change the **Name** of the interface after you click **Save**.
 
-3. (Optional but recommended) Enter any notes or reminders about this particular bridge in the **Description** field.
+4. (Optional but recommended) Enter any notes or reminders about this particular bridge in the **Description** field.
+
+5. Select the interfaces on the **Bridge Members** dropdown list.
 
-4. Select the interfaces on the **Bridge Members** dropdown list.
+6. Next to **Aliases** click **Add** to enter the IP address for this bridge interface. (Optional) click **Add** to display an additional IP address field for each additional IP address you want to add.
 
-5. Next to **Aliases** click **Add** to enter the IP address for this bridge interface. (Optional) click **Add** to display an additional IP address field for each additional IP address you want to add.
- 
 ![AddInterfaceNicDeviceOtherSettingsSCALE](/images/SCALE/22.12/AddInterfaceNicDeviceOtherSettingsSCALE.png "Add IP and Subnet Mask")
 
-6. Click **Save** when finished. You should see the bridge name you created in **Interfaces** with its associated IP address information.
+7. Click **Save** when finished. You should see the bridge name you created in **Interfaces** with its associated IP address information.
```

Comment:
I know this is legacy content but something bugs me about this sentence. so I suggest this as a change. What do you think?
```suggestion
7. Click **Save** when finished. The bridge name created shows in **Interfaces** with its associated IP address information.
```